### PR TITLE
add retry interval (base_interval, max_interval)

### PR DESCRIPTION
**Pull request by niqniqniqq**
1. Added envoy-filter for retry interval.
2. Added description of retry interval to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1049,6 +1050,61 @@ kubectl logs <v1 pod> -n testns1 -c istio-proxy -f --tail 0
 [2020-11-23T10:52:12.711Z] "GET /delay/15 HTTP/1.1" 0 DC "-" "-" 0 0 1999 - "-" "curl/7.35.0" "fc786864-ce72-4eae-9e20-9d9df1ac26ac" "httpbin.testns1.svc.cluster.local" "127.0.0.1:80" inbound|8000|http|httpbin.testns1.svc.cluster.local 127.0.0.1:45190 192.168.94.99:80 192.168.64.239:55220 outbound_.8000_.v1_.httpbin.testns1.svc.cluster.local default
 ```
 
+#### retries (baseInterval and maxInterval)
+The retry interval can be controlled by setting envoyfilter.
+In the following configuration, base_interval is set to 4S and max_interval is set to 5s.
```
Comment (hiroyuki-kamei) [inline README.md:1055]: In the following configuration, base_interval is set to 4s and max_interval is set to 5s.

**Review by hiroyuki-kamei — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chinkame) [inline README.md]: It seems to be the type. Please fix it.
```suggestion
In the following configuration, base_interval is set to 4s and max_interval is set to 5s.
```

Comment (niqniqniqq) [reply README.md]: @hiroyuki-kamei 
Fixed.

File: README.md [hunk 2]
```diff
@@ -1049,6 +1050,61 @@ kubectl logs <v1 pod> -n testns1 -c istio-proxy -f --tail 0
...
+Send the HTTP request to http://httpbin.testns1.svc.cluster.local/status/504. The endpoint is supposed to return a status 504 response.
+
+```bash
+export NAMESPACE=testns1
+export SLEEP_POD=$(kubectl get pod -l app=sleep -n $NAMESPACE -o jsonpath={.items..metadata.name})
+kubectl exec "${SLEEP_POD}" -c sleep -n $NAMESPACE -- curl -sv  "http://httpbin.${NAMESPACE}.svc.cluster.local/status/504"
+```
+
+You can check the access log for retries with 5s as the max retry interval.
+The retry interval is determined by the Envoy retries Algorithm described above.
```
Comment (akitok) [inline README.md]: It is better to reverse these two sentences and improve the context to sample access log.
```suggestion
The retry interval is determined by the Envoy retries Algorithm described above.
You can check the access log for retries with the max retry interval of 5 second as follows:
```

**Comment by akitok** [comment]
@niqniqniqq 
This is all comment from me. 
Please check it. Good commit!

**Review by chinkame — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (niqniqniqq) [reply README.md]: @akitok 
Fixed.

**Review by akitok — APPROVED** [review]
Looks good to me. Thanks for your nice commit!

**Pull request merged**